satellite/metainfo/metainfo.go

```diff
@@ -43,26 +44,28 @@ type APIKeys interface {
 
 // Endpoint metainfo endpoint
 type Endpoint struct {
-	log           *zap.Logger
-	metainfo      *Service
-	orders        *orders.Service
-	cache         *overlay.Cache
-	apiKeys       APIKeys
-	accountingDB  accounting.DB
-	maxAlphaUsage memory.Size
+	log            *zap.Logger
+	metainfo       *Service
+	orders         *orders.Service
+	cache          *overlay.Cache
+	apiKeys        APIKeys
+	accountingDB   accounting.DB
+	liveAccounting live.Service
+	maxAlphaUsage  memory.Size
 }
 
 // NewEndpoint creates new metainfo endpoint instance
-func NewEndpoint(log *zap.Logger, metainfo *Service, orders *orders.Service, cache *overlay.Cache, apiKeys APIKeys, acctDB accounting.DB, maxAlphaUsage memory.Size) *Endpoint {
+func NewEndpoint(log *zap.Logger, metainfo *Service, orders *orders.Service, cache *overlay.Cache, apiKeys APIKeys, acctDB accounting.DB, liveAccounting live.Service, maxAlphaUsage memory.Size) *Endpoint {
 	// TODO do something with too many params
 	return &Endpoint{
-		log:           log,
-		metainfo:      metainfo,
-		orders:        orders,
-		cache:         cache,
-		apiKeys:       apiKeys,
-		accountingDB:  acctDB,
-		maxAlphaUsage: maxAlphaUsage,
+		log:            log,
+		metainfo:       metainfo,
+		orders:         orders,
+		cache:          cache,
+		apiKeys:        apiKeys,
+		accountingDB:   acctDB,
+		liveAccounting: liveAccounting,
+		maxAlphaUsage:  maxAlphaUsage,
```

Comment:
there are a lot of places still using `MaxAlphaUsage` variables.  Should all that "alpha" stuff be removed since this project limiting isn't related to the alpha release anymore? Or should we create a different ticket to clean all that up?

Reply:
@iglesiasbrandon told me it still is going to be limited to the alpha release after all, so I put back all the “alpha” references I had taken out previously. See 0a8012420143bca9e987cf9458db7c9512fa6fa8

Reply:
odd. @jtolds told me that it wasn't limited to alpha anymote. hmmm

Reply:
I tried to follow up more, but didn't get answers, so I decided to leave it all intact for now (and avoid requiring a change in the satellite config files, because they have the "alpha" in there too). If we decide to keep it, we can remove the alpha name down the road, I guess.